packages/shared/lib/services/sync/on-event-scripts.service.ts

```diff
@@ -77,10 +77,10 @@ export const postConnectionScriptService = {
                     });
                 }
             }
-            await trx.insert(postConnectionInserts).into(TABLE);
+            await trx.insert(onEventInserts).into(TABLE);
         });
     },
-    getByConfig: async (configId: number): Promise<PostConnectionScript[]> => {
-        return db.knex.from<PostConnectionScript>(TABLE).where({ config_id: configId, active: true });
+    getByConfig: async (configId: number): Promise<OnEventScript[]> => {
+        return db.knex.from<OnEventScript>(TABLE).where({ config_id: configId, active: true });
```

Comment:
For now all rows are related to connection creation events. In following PR I will add a `eventType` parameter